js/index.js

```diff
@@ -57,27 +57,41 @@ document.addEventListener("DOMContentLoaded", () => {
         ************ PROJECTS  *************
         ====================================
      */
-  const githubRequest = new XMLHttpRequest();
-
-  githubRequest.addEventListener("load", function () {
-    if (githubRequest.readyState === 4 && githubRequest.status === 200) {
-      let repositories = JSON.parse(this.response);
-      const projectSection = document.getElementById("projects");
-      const projectList = projectSection.querySelector("ul");
-
-      for (let i = 0; i < repositories.length; i++) {
-        const project = document.createElement("li");
-        const repoLink = document.createElement("a");
-        repoLink.href = repositories[i].html_url;
-        repoLink.textContent = repositories[i].name;
-
-        project.appendChild(repoLink);
-        projectList.appendChild(project);
-      }
+  const projectSection = document.getElementById("projects");
+  const projectList = projectSection.querySelector("ul");
+  const url = "https://api.github.com/users/wessstt/repos";
+
+  fetch(url)
+    .then(checkStatus)
+    .then(parseJSON)
+    .then(githubRepo)
+    .catch((error) =>
+      console.log("Oops, there was an error fetching your request.", error)
+    );
```

Comment:
```suggestion
    .catch((error) => {
      console.log("Oops, there was an error fetching your request.", error)
    });
```
Slight preference: since this is a statement, and not just an expression, I'd enclose in curly braces.